src/System.Private.CoreLib/shared/System/Reflection/AssemblyName.cs

```diff
@@ -172,7 +172,7 @@ public void SetPublicKey(byte[]? publicKey)
 
         // The compressed version of the public key formed from a truncated hash.
         // Will throw a SecurityException if _publicKey is invalid
-        public byte[] GetPublicKeyToken()
+        public byte[]? GetPublicKeyToken()
         {
             if (_publicKeyToken == null)
                 _publicKeyToken = ComputePublicKeyToken();
```

Comment:
Nit: The return value of `ComputePublicKeyToken` should by `byte[]?`

Reply:
Fixed